concordium-sdk/src/main/java/com/concordium/sdk/ClientV2.java

```diff
@@ -161,6 +162,18 @@ public ConsensusStatus getConsensusInfo() {
         return to(grpcOutput);
     }
 
+    /**
+     * Retrieves a {@link BlockInfo}
+     *
+     * @param blockHashInput the block {@link BlockHashInput} to query.
+     * @return A {@link BlockInfo} for the block
+     */
```

Comment:
Please state what will happen if an "unknown" (a block that the node does not know about) block is being queried via the 'GIVEN' constructor.

Reply:
Right now it throws StatusRuntimeException, Do we need to change this behaviour? 
<img width="864" alt="Screenshot 2023-04-02 at 7 38 21 PM" src="https://user-images.githubusercontent.com/32020192/229358089-4f7a73ab-9e82-4c90-b754-2133e0c19f0b.png">